toolkits/global/packages/global-article/__tests__/authors.js

```diff
@@ -22,13 +20,13 @@ describe('Authors', () => {
 		let listChildren = '';
 
 		for (let i = 0; i < number; i++) {
-			listChildren += `<li><a href="#author${i}">author${i}</a><sup><a href="#Aff${i}" class="js-no-scroll" tabindex="-1">1</a>
-				${separator ? `<span class="js-separator">,</span>` : ''}
+			listChildren += `<li><a href="#author${i}">author${i}</a><sup><a href="#Aff${i}" tabindex="-1" data-author-popup="author${i}">1</a>
```

Comment:
I think this would be cleaner if it was creating the element and appending it to a fragment rather than creating a string. Innerhtml is pretty slow I think

Reply:
Yeah tbh this entire file needs refactoring in to es6 and improving in many places. I'll do this though as a nice little tidy up step

Reply:
Oh wait, I thought this comment was about authors.js but it's about the test for authors.js. Did you mean to post it about the module authors.js? Because that's a legit comment about the use of `.innerHTML =` in that file. But I guess in terms of the test file it doesn't matter how efficiently the dom setup logic is, our laptops are fast enough that the difference with be in perceivable?

Reply:
Oh yeah sorry, my mistake. Didn't realise it was a test file

Reply:
But the funny thing is, you make a good point about the actual module! haha